velox/exec/fuzzer/AggregationFuzzer.cpp

```diff
@@ -367,7 +391,17 @@ void AggregationFuzzer::go() {
             (signature.name.find("approx_") == std::string::npos) &&
             vectorFuzzer_.coinToss(0.2);
 
-        auto call = makeFunctionCall(signature.name, argNames, sortedInputs);
+        // Exclude approx_xxx, merge aggregations since it conflicts with
```

Comment:
For approx_, say, approx_distinct is different with count(distinct), there may be no need to test something like `approx_distinct(distinct x)`.
For merge, it is a HyperLogLog functions, whose input type is HyperLogLog that is not supported by `DistinctAggregations`.

Reply:
There may be more unsupported distinct aggs, so I add a TODO to add a exluce/allow list in the future (this is why I only hardcode count distinct and array_agg at the first version).

Reply:
> For merge, it is a HyperLogLog functions, whose input type is HyperLogLog that is not supported by DistinctAggregations.

This can be handled by supportsDistinctInputs check, no?

> For approx_, say, approx_distinct is different with count(distinct), there may be no need to test something like approx_distinct(distinct x).

It may not be very meaningful, but still should work. Hence, it is good to test.

In the future, it would be nice to extend Aggregate functions metadata to allow specifying whether a function is sensitive to the order of inputs and presence of duplicate values. Then, Velox can drop 'order by' and 'distinct' when these are not necessary and Fuzzer can skip testing such functions. CC: @kagamiori

Reply:
>This can be handled by supportsDistinctInputs check, no?

Add a type check in `supportsDistinctInputs`?
```C++
if (!arg->isComparable() || arg == HYPERLOGLOG()) {
      return false;
}
```

Reply:
That should work.